city_scrapers/spiders/il_sex_offender_management.py

```diff
@@ -0,0 +1,249 @@
+import json
+import re
+from datetime import datetime, timedelta
+from io import BytesIO, StringIO
+
+import scrapy
+from city_scrapers_core.constants import BOARD, CANCELLED, PASSED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+
+
+class IlSexOffenderManagementSpider(CityScrapersSpider):
+    name = "il_sex_offender_management"
+    agency = "Illinois Sex Offender Management Board"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        "https://www2.illinois.gov/idoc/" + "Pages/SexOffenderManagementBoard.aspx"
```

Comment:
Is there a reason for the "+" here? If the URL is longer than the allowed character limit it's usually more readable to add `#noqa` at the end of the line so that it doesn't trigger errors with `flake8` or `black` to override the checks there